### PR TITLE
feat(EMS-1046): Accessibility statement

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR adds the accessibility statement page.

## Changes

- Setup route, controller and template.
- E2E test coverage.

### CODE REVIEW

File: e2e-tests/cypress/e2e/journeys/accessibility-statement.spec.js [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+    cy.login();
+
+    partials.footer.supportLinks.accessibilityStatement().click();
+    cy.url().should('include', ROUTES.ACCESSIBILITY_STATEMENT);
+
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('exip-session');
+  });
+
+  it('passes the audits', () => {
```
Comment (Zainzzkk) [inline e2e-tests/cypress/e2e/journeys/accessibility-statement.spec.js:43]: are we not using the new cy command for this?

File: e2e-tests/cypress/e2e/journeys/accessibility-statement.spec.js [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+    accessibilityStatementPage.serviceLink().invoke('text').then((text) => {
+      expect(text.trim()).equal(SERVICE_LINK.TEXT);
+    });
+
+    accessibilityStatementPage.serviceLink().should('have.attr', 'href', SERVICE_LINK.HREF);
+  });
+
+  describe('using our service', () => {
+    it('renders a heading', () => {
+      usingOurService.heading().invoke('text').then((text) => {
```
Comment (Zainzzkk) [inline e2e-tests/cypress/e2e/journeys/accessibility-statement.spec.js:87]: checkText?

Comment (ttbarnes) [reply e2e-tests/cypress/e2e/journeys/accessibility-statement.spec.js:87]: Same as above

**Review by Zainzzkk — APPROVED** [review]
lgtm

**Pull request merged**